### DISCUSSION TITLE
Outputting all the poses

### DISCUSSION THREAD

**Discussion by MuhammadAttaElhamouly** [RosettaCommons/rosetta#256] [RosettaScripts] [answered]
I connected PyMOL to Rosetta and sent many poses to follow up on the protocol progress, but this process needs PyMOL to be online and connected to Rosetta and the final output files will have only the final pose, I'm wondering if there is another way or part I can add to Rosettascript to make the final output have all the poses I requested

**Comment by roccomoretti** [comment, marked as answer]
There's a number of [Reporting/Saving movers](https://docs.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/Movers/Movers-RosettaScripts#meta-movers_reporting-saving) in RosettaScripts.

Something like the [PDBTrajectoryRecorder](https://docs.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/Movers/movers_pages/PDBTrajectoryRecorderMover) or [ReportToDB](https://docs.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/Movers/movers_pages/ReportToDBMover) may be what you're looking for. (The [DumpPdb mover](https://docs.rosettacommons.org/docs/latest/scripting_documentation/RosettaScripts/Movers/movers_pages/DumpPdbMover) is probably too simplistic for your purposes, though with the "tag_time" option, it may work.)

**Reply by MuhammadAttaElhamouly** [reply]
Many thanks
